karma.conf.js

```diff
@@ -20,7 +20,9 @@ module.exports = function(config) {
       'node_modules/jasmine-promises/dist/jasmine-promises.js',
       'node_modules/webcomponentsjs/full.js',
       'dist/js/patternfly.js',
-      'src/**/*.spec.js'
+      'src/**/*.spec.js',
+      'dist/css/patternfly.css',
+      'dist/css/patternfly-webcomponents.css'
```

Comment:
Please do not include CSS files here. PhantomJS goes out of memory when you try to load CSS and WebFonts in Travis.